daemons/dss-scalability-test/domovoilib/json_generator/__init__.py

```diff
@@ -0,0 +1,81 @@
+import json
```

Comment:
Is this the real json generator for scaletest, or a stand-in until the code that @Bento007 is working on lands?

If it's the real thing, I would prefer it if it was broken out into its own PR, and probably moved to dss.util.

Reply:
it's a standin